data/people/affiliations.tsv

```diff
@@ -2,136 +2,139 @@ affiliation	address	city	country	longitude	latitude
 Amsterdam UMC, locatie VUmc	Amsterdam UMC locatie AMC, 9, Meibergdreef, Zuidoost, Amsterdam, Noord-Holland, Nederland, 1105 AZ, Nederland	Amsterdam	Nederland	4.956576595884709	52.2924635
 Amsterdam University Medical Centers, Radiology and Nuclear Department, Amsterdam, 1013 EG, The Netherlands	Amsterdam, Noord-Holland, Nederland	Amsterdam	Nederland	4.8924534	52.3730796
 Amsterdam University Medical Centers, Radiology and Nuclear Medicine, Amsterdam, 1081 HV, The Netherlands	Amsterdam, Noord-Holland, Nederland	Amsterdam	Nederland	4.8924534	52.3730796
-Boston Children's Hospital, Department of Neurology, Boston, 02115, United States of America	Boston, Suffolk County, Massachusetts, United States	Boston	United States	-71.060511	42.3554334
-Boston University, Biomedical Engineering, Boston, 02215, United States of America	Boston, Suffolk County, Massachusetts, United States	Boston	United States	-71.060511	42.3554334
+Boston Children's Hospital, Department of Neurology, Boston, MA, 02115, USA	Maroc ⵍⵎⵖⵔⵉⴱ المغرب		Maroc ⵍⵎⵖⵔⵉⴱ المغرب	-7.3362482	31.1728205
```

Comment:
I think this might not be the change you wanted.

```suggestion
Boston Children's Hospital, Department of Neurology, Boston, 02115, United States of America	Boston, Suffolk County, Massachusetts, United States	Boston	United States	-71.060511	42.3554334
```

Reply:
Also not sure what formatting you're choosing for this, but I'm assuming you want it all in the same language.